src/parser.cpp

```diff
@@ -1323,7 +1323,7 @@ const Statement *Parser::parseReturnStatement(Scope *scope, int line)
 const Statement *Parser::parseVarStatement(Scope *scope, int line)
 {
     ++i;
-    std::vector<const ReferenceExpression *> varrefs;
+    std::map<std::string, Variable *> variables;
```

Comment:
I'd like to see this remain a vector. The reason is, the order of items in a map is not consistent. This could cause the same source file, when compiled more than once, to result in different bytecode (especially if built on a different machine). You can still access the name of the variable with `v->name` (for the `addName` calls below).